src/ir/function.rs

```diff
@@ -130,7 +130,14 @@ fn get_abi(cc: CXCallingConv) -> Abi {
 
 fn mangling_hack_if_needed(ctx: &BindgenContext, symbol: &mut String) {
     if ctx.needs_mangling_hack() {
-        symbol.remove(0);
+        match symbol.chars().next().unwrap() {
+            // Stripping leading underscore for all names on Darwin and
+            // C linkage functions on Win32.
+            '_' => { symbol.remove(0); }
```

Comment:
No, it needs. `String::remove` returns `char` and `String::insert` returns nothing.